src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -57,6 +70,17 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
 
     private activeFoci: Focus[] | undefined;
 
+    private encryptMedia = false;
+    private _encryptionKeys = new Map<{ userId: string; deviceId: string }, string>();
+
+    /**
+     * A map of keys used to encrypt and decrypt (we are using a symmetric
+     * cipher) given participant's media. This also includes our own key
+     */
+    public get encryptionKeys(): Map<{ userId: string; deviceId: string }, string> {
```

Comment:
Also I've just realised that this is a Map with objects as keys. Beware that  this works because the only way it's being read is by iterating over the whole Map and getting the values out. `encryptionKeys.get({userId: foo, deviceId: bar})` won't work unless you actually pass in a reference to the same object (ie. equal according to SameValueZero equality). This definitely feels like a confusing data structure. If we['re only ever going to iterate over the whole thing, I'd suggest just using an array. If we expect to actually look up the keys in the map, I'd use a map with strings as keys.